pkg/kdump/kdump-view.jsx

```diff
@@ -263,12 +265,39 @@ export class KdumpPage extends React.Component {
     }
 
     handleTestSettingsClick() {
+        // if we have multiple targets defined, the config is invalid
+        const target = this.props.kdumpStatus.target;
+        let verifyMessage;
+        if (!target.multipleTargets) {
+            let path = target.path || "/var/crash";
+            if (target.type === "local") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be stored in $0 as $1, if kdump is properly configured."),
+                    <span className="pf-v5-u-font-family-monospace-vf">{path}</span>,
+                    <span className="pf-v5-u-font-family-monospace-vf">vmcore</span>);
+            } else if (target.type === "ssh" || target.type == "nfs") {
+                path = (target.type == "nfs" && path[0] !== '/') ? '/' + path : path;
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be copied through $0 to $1 as $2, if kdump is properly configured."),
+                    <span className="pf-v5-u-font-family-monospace-vf">{target.type === "ssh" ? "SSH" : "NFS"}</span>,
+                    <span className="pf-v5-u-font-family-monospace-vf">{`${target.server}:${target.type === "nfs" ? target.export + path : path}`}</span>,
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19107-20230823-065815-45cdb56e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)